.travis.yml

```diff
@@ -5,24 +5,48 @@ env:
   global:
     - NPM_CONFIG_LOGLEVEL=warn
     - secure: lpcZNywBxi9I7Zy8wpgq4y5cMT3WJhRuswT/AEILAoKKD4utAH1ibLigjcYTjXai2oHP8hH8e2BBvAibJtfWVlFUaaXWk1RrR7Zs8ligdpQkmKOUB7m41xj2JiYFn3DNntAd21Gvu8zq+SlY2TG50VjQmWOkVf49Yq6/ED3nQ6r352dV6II0e+Qm99+VA9em1QYGLI2jiAZVpgMXOCw3jHrtusVMAyrwPHixrpw1TvoflvHTMLlAxeaa4MyBxy6+qBEjacXxQvi8+OGYNyi7zLYxijTUMqJwVB8L9uLgrUQbWP8HA3QzCnKAim/SADsgoMPYxWBQk45mdgJptznZVrQON4gL222LZrSuuwGK4OVu132hrDRHnKJ8bQS95WvpQkPTE6Yh3rDAuAMprrEIJt4c4WhyXovo9165aG03AqkqeSIQOmCXPFld2+qzqnI+V+QsocDhtsz29kCEHjjLUbxg0GzW8JfBFSjVW38FebvPfz8dOhj1osNIIjKxi8Twdtx8knfuHkEb6q2Fp3nvDhUPai/1FXdiNo+/Jz1FcPvDuAaa7hCj1ZRos5Ce/iJJwndgal5PLFdr5lwkm2Jd7iYYwv7tPStFSX42tuSf92LZns/hHDGTI439eu3/lryCoByG/+98qa+iIRmjLyesbfn0+0PQMjSfyo3Gwe8rtnQ=
-before_install:
-  - npm install -g greenkeeper-lockfile@1
-before_script: greenkeeper-lockfile-update
-after_script: greenkeeper-lockfile-upload
+
 install:
-  - npm install
-  - npm run build
-script:
-  - set -e
-  - ./scripts/check-pristine-state package-lock.json
-  - npm run danger
-  - npm test
-  - npm run docs:dist
-  - if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then npm run storybook:dist; fi
-  - test -e dist/index.html
-  - test -e dist/sassdoc/index.html
+  echo 'Backpacking, hang on!'
+
+jobs:
+  include:
+    - stage: install dependencies
+      script:
+        - npm install -g greenkeeper-lockfile@1
+        - greenkeeper-lockfile-update
+        - npm install
+        - npm run build
+        - greenkeeper-lockfile-upload
+
+    - stage: shared cache
+      script:
+        - npm run bootstrap
```

Comment:
`npm run bootstrap` is happening 4 times here ... that cant be right?

Reply:
This was mentioned as an issue in the PR description:

> The only problem I have so far is I have to run npm run bootstrap in every step to have lerna happy and I can't test a merge onto master so that might break

Reply:
just to double check I'm work, this is a build without the bootstrap step
https://travis-ci.org/Skyscanner/backpack/jobs/400023027
lerna is not happy 😄

Reply:
bootstrap runs as post install hook so it does run once, however it creates symlinks and I don't think those are preserved between steps